src/openslide-vendor-dicom.c

```diff
@@ -711,6 +726,76 @@ static void add_properties(openslide_t *osr, struct dicom_level *level0) {
   }
 }
 
+static bool quickhash_level_pixels(struct _openslide_hash *hash,
+                                   struct dicom_level *level) {
+  double d;
+  if (!get_tag_decimal_str(level->file->metadata, NumberOfFrames, 0, &d)) {
+    return false;
+  }
+  uint32_t num_frames = d;
+
+  int64_t total_bytes = 0;
+  for (uint32_t i = 0; i < num_frames; i++) {
+    DcmFrame *frame = dcm_filehandle_read_frame(NULL,
+                                                level->file->filehandle,
+                                                i + 1);
+    if (frame == NULL) {
+      return false;
```

Comment:
Likewise, should this fail the open?

Reply:
I don't think this can realistically fail, so making it just disable quickhash seems reasonable. It'd very probably fail on get tile  soon after, so I don't think the error (eg. a corrupt or badly malformed file, perhaps?) would be hidden.

Reply:
I made open fail if frame get in quickhash computation fails.